ui/partials/perf/alertsctrl.html

```diff
@@ -115,11 +115,15 @@
           &nbsp;(<span ng-class="{'alert-invalid': alert.isInvalid(), 'alert-untriaged': alert.isUntriaged()}">{{alert.getStatusText()}}</span><span ng-show="alert.related_summary_id">
             <!-- Reassigned or downstream *to* another alert -->
             <span ng-if="alert.related_summary_id !== alertSummary.id">
-              to <a href="#/alerts?id={{alert.related_summary_id}}" target="_blank">alert #{{alert.related_summary_id}}</a>
+              to <a href="#/alerts?id={{alert.related_summary_id}}" target="_blank"
+                    uib-tooltip="{{alert.classifier ? 'Classified by ' + alert.classifier : 'Classified automatically'}}"
```

Comment:
Move this tooltip (with the updated logic I mentioned in the bug to not say "untriaged" alerts are autoclassified) up to the span just below the "alert-title" td.